picky/src/http/http_signature.rs

```diff
@@ -229,8 +252,11 @@ impl FromStr for HttpSignature {
         for item in items {
             if let Some(index) = item.find('=') {
                 let (key, value) = item.split_at(index);
-                if value.len() >= 3 {
-                    keys.insert(key.trim(), value[2..value.len() - 1].to_owned());
+                let value = value[1..].trim();
+                if value.starts_with('"') && value.len() >= 2 {
```

Comment:
The function trim_matches (or trim_start_matches and trim_end_matches) could be used instead of doing an if. But both work.

Reply:
Nice catch.